nexus/db-queries/src/db/datastore/inventory.rs

```diff
@@ -469,6 +491,128 @@ impl DataStore {
                 }
             }
 
+            // Insert rows for the root of trust pages that we found. This is
+            // almost identical to inserting cabooses above, and just like for
+            // cabooses, we do this using INSERT INTO ... SELECT. We have these
+            // three tables:
```

Comment:
I'd probably avoid duplicating the existing comment and just say "it looks like the other case".  It's not a big deal.  It just makes it look to me like there's something more special about this case than [I think] there is.